### ISSUE TITLE
a11y team sync - 20th March 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/czi-scientific-python-mgmt#66]
@smeragoel 
	- Working on colours for SciPy (see https://github.com/scipy/scipy/issues/19958) need to sync with @melissawm 
	- Related https://github.com/numpy/numpy/issues/25927 
	- Look back into collapsible sidebar https://github.com/pydata/pydata-sphinx-theme/issues/1072
	- Start looking at SciPython-specific components in https://github.com/scientific-python/scientific-python-hugo-theme/issues/477

@gabalafou 	
	- Next work on https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/50
	- Look into automating screenshots for accessible pygments https://github.com/simonw/shot-scraper
	- Example on adding a ref to a card https://github.com/microsoft/nni/blob/767ed7f22e1e588ce76cbbecb6c6a4a76a309805/docs/extension/cardlinkitem.py#L10

**Comment by melissawm** [comment]
See also https://github.com/numpy/numpy/issues/26058

**Comment by smeragoel** [comment]
For [#19958](https://github.com/scipy/scipy/issues/19958), I synced with @melissawm and this is the summary of our discussion:

- We will be sticking to the PST colour palette, and remove all colour customisations (not just a11y violations). 
- @melissawm will create a PR with the custom css file removed, and I'll check for a11y violations and any deviations from the PST colour palette.
